app/not-found.tsx

```diff
@@ -4,7 +4,7 @@ import Section from "@packages/components/Section";
 
 export default function PageNotFound() {
   return (
-    <Section pb="0" pt="8rem">
+    <Section className="p-0 pt-32">
```

Comment:
Shoudn't it be `pb-0` or `px-0`?
```suggestion
    <Section className="pb-0 pt-32">
```